static/src/js/components/Map/Map.tsx

```diff
@@ -79,12 +88,24 @@ import { eventEmitter } from '../../events/events'
 
 import 'ol/ol.css'
 import './Map.scss'
-
-let previousGranulesKey
-let previousProjectionCode
+import {
+  GranuleMetadata,
+  MapGranule,
+  ProjectionCode,
+  Query,
+  Shapefile,
+  SpatialSearch
+} from '../../types/sharedTypes'
+import { Colormap } from '../Legend/Legend'
+import { MapView } from '../../zustand/types'
+
+let previousGranulesKey: string
+let previousProjectionCode: ProjectionCode
 let layersAdded = false
 
 // Render the times icon to an SVG string for use in the focused granule overlay
+// Disable a testing-library rule because this isn't a test
+// eslint-disable-next-line testing-library/render-result-naming-convention
```

Comment:
I'm fine for this now this diff is already large enough haha. Seems like something on the linter must be misconfigured would not this this should be possible based on the filetype